File: app/src/main/java/com/example/msdc/api/CreditResponse.java
package com.example.msdc.api;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class CreditResponse {
    @SerializedName("cast")
    private final List<CreditCastResult> cast = new ArrayList<>();

    @SerializedName("crew")
    private final List<CreditCrewResult> crew = new ArrayList<>();

    public List<CreditCastResult> getCast() {
        return cast;
    }

    public List<CreditCrewResult> getCrew() {
        return crew;
    }
}

File: app/src/main/java/com/example/msdc/api/CreditResponse.kt
package com.example.msdc.api

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class CreditResponse(
    @field:SerializedName("cast") val cast: ArrayList<CreditCastResult>? = null,
    @field:SerializedName("crew") val crew: ArrayList<CreditCrewResult>? = null
) : Parcelable